ansible/roles/common_installs/tasks/main.yml

```diff
@@ -92,7 +93,6 @@
   pip: name={{ item }}
   sudo: yes
   with_items:
-    - fabric
```

Comment:
ah! that diff shows the deploy script `fabfile.py`. i didn't know we used to keep that in here. guessing that's where that requirement came from. based on that i think we can actually get rid of `sh` too, not too big of a deal though